.ci-builds/data-filter.json

```diff
@@ -8,16 +8,18 @@
     ]
   },
 // Test mixed feature filter and resource filter
+// Exlude translit data so we can run that test
```

Comment:
minor nit: maybe refer to the ticket number (ICU-20673) instead of just "that" so there's a breadcrumb to follow later on for somebody else?  :)